kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java

```diff
@@ -64,17 +70,22 @@ void createSchemaAvro() {
     @Order(2)
     void updateSchemaAvro() {
         AVRO_API.setValuePath(System.getProperty("user.dir") + "/src/main/resources/testData/schema_avro_for_update.json");
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        schemaRegistryList.openSchema(AVRO_API.getName())
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
+        schemaRegistryList
+                .waitUntilScreenReady()
+                .openSchema(AVRO_API.getName());
+        schemaDetails
                 .waitUntilScreenReady()
                 .openEditSchema();
```

Comment:
after openEditSchema() we should wait schemaCreateForm

Reply:
Added